akka-http2-support/src/main/scala/akka/http/impl/engine/http2/FrameEvent.scala

```diff
@@ -14,6 +14,7 @@ sealed trait StreamFrameEvent extends FrameEvent {
   def streamId: Int
 }
 
+final case class GoAwayFrame(lastStreamId: Int, errorCode: ErrorCode, debug: Option[String] = None) extends FrameEvent
```

Comment:
Hmm... Why would someone put sensitive data in there? If we go by the logic that someone could send us sensitive data in any kind of field we couldn't log anything.

In cases where *we* want to log the GoAwayFrame we also want to see its contents so it doesn't seem useful for me to hide it here. Does anyone know the logic behind that paragraph in the spec?